packages/ui-library/src/elements/text-input/stories.js

```diff
@@ -15,20 +15,18 @@ export default {
 };
 
 export const Factory = {
-	component: ( args ) => <StoryComponent { ...args } />,
 	parameters: {
 		controls: { disable: false },
 	},
 };
 
-export const DatePicker = StoryComponent.bind( {} );
-
-DatePicker.parameters = {
-	controls: { disable: false },
-};
-
-DatePicker.args = {
-	type: "date",
+export const DatePicker = {
```

Comment:
About the weird `dd-----yyyy`, we checked, and that is a browser + operating system thing. Apparently Chrome on Windows reads from your system date settings and uses that there. So that is actually out of our control and it was the point of removing any override we did unrelated to this PR.

Removing the `placeholder` from the story: we don't do anything with the `placeholder` anymore, and most environments don't seem to support changing it (expect Windows Chrome apparently 😉)